CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.9.0
+
+* Added JWT authentication
+* Added Batch requesting to http client
+
```

Comment:
Could i ask you to include my 0.8.2 changes here with the JWT auth? i didn;t think of it in my PR >_<